requirements-test.txt

```
@@ -5,18 +5,20 @@ cfgv==3.4.0
 coverage==7.6.0
 decorator==5.1.1
 distlib==0.3.8
-exceptiongroup==1.2.1
+exceptiongroup==1.2.2
 executing==2.0.1
 filelock==3.15.4
 flake8==7.1.0
 identify==2.6.0
+idna==3.7
 iniconfig==2.0.0
 ipdb==0.13.13
-ipython==8.26.0
+ipython==8.18.1
```

Weird. We really don't care. We only use `ipython` when doing interactive debugging.
During the CI/CD build, it doesn't even need to be there.

I'm going to bump it down, and see if the build can continue.

```suggestion
ipython==8.13.0
```